android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -205,6 +205,10 @@ public void onZoomSDKInitializeResult(int errorCode, int internalErrorCode) {
     }
   }
 
+  @Override
+  public void onZoomAuthIdentityExpired(){
+  }
+
```

Comment:
why is this needed?

Reply:
Version of lib files :v5.0.24437.0708
onZoomAuthIdentityExpired is a required abstract method now, without it it simply won't build.

Basically what i did here was took your library, replace the lib files with new ones downloaded from zoom's repo (zoom-sdk-android-master) and then proceeded to make all the changes so that it'll build.